### PR TITLE
Update README for Voting Status on many and add BSIP78, 79

### PR DESCRIPTION
**Pull request by ryanRfox**
BBF has draft WP ready for voting on most of the the "done" BSIPs already and will finalize the last few today. BSIP78 is newly added.
See "drafts" tab for current list: https://www.bitshares.foundation/worker/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,34 +52,36 @@ Number             | Title                                                    |
...
 [44](bsip-0044.md) | Hashed Time-Locked Contract                                                 | Ryan R. Fox | Protocol | Installed
-[45](bsip-0045.md) | Add bitAsset Backing Collateral flag/permission                             | Customminer | Protocol | Draft
+[45](bsip-0045.md) | Add bitAsset Backing Collateral flag/permission                             | Customminer | Protocol | Voting
 [46](https://github.com/bitshares/bsips/pull/111) | Escrow Concepts                              | Taconator | Informational | Accepted
-[47](bsip-0047.md) | Vote Proxies for Different Referendum Categories and explicit voting operation                 | Fabian Schuh | Protocol | Draft
-[48](https://github.com/bitshares/bsips/pull/115) | Add Flag to Asset to Prevent Manipulating Max Supply | Fabian Schuh | Protocol | Draft
+[47](bsip-0047.md) | Vote Proxies for Different Referendum Categories and explicit voting operation                 | Fabian Schuh | Protocol | Voting
+[48](https://github.com/bitshares/bsips/pull/115) | Add Flag to Asset to Prevent Manipulating Max Supply | Fabian Schuh | Protocol | Voting
```
Comment (abitmore) [inline README.md]: Already voting? Please merge the draft into master. If already done, please update the link.

File: README.md [hunk 2]
```diff
@@ -52,34 +52,36 @@ Number             | Title                                                    |
...
+[57](bsip-0057.md) | Managed Vesting Balances | Blockchain Projects BV | Protocol | Voting
 [58](bsip-0058.md) | Global Settlement Protection Through Price Feeding | Jerry Liu | Informational | Accepted
 [59](bsip-0059.md) | Adjustment of MSSR and MCR Through Voting | Jerry Liu | Informational | Accepted
 [60](bsip-0060.md) | BitShares URI scheme | John Titor, Stefan Schießl, Abit More | Informational (Client Protocol) | Accepted
-[61](https://github.com/bitshares/bsips/issues/150) | Operation to Update Limit Orders | Nathan Hourt | Protocol | Draft
-[62](https://github.com/bitshares/bsips/issues/156) | Close Short Position | Stefan Schießl | Protocol | Draft
-[63](bsip-0063.md) | Short-lived Unidirectional Payment Channels | Christopher J. Sanborn | Informational | Draft
-[64](bsip-0064.md) | Optional HTLC preimage length, HASH160 addition, and memo field | John Jones, Abit More | Protocol | Draft
+[61](https://github.com/bitshares/bsips/issues/150) | Operation to Update Limit Orders | Nathan Hourt | Protocol | Voting
+[62](https://github.com/bitshares/bsips/issues/156) | Close Short Position | Stefan Schießl | Protocol | Voting
```
Comment (abitmore) [inline README.md]: Already voting? Please merge the draft into master. If already done, please update the link.

File: README.md [hunk 3]
```diff
@@ -52,34 +52,36 @@ Number             | Title                                                    |
...
-[73](bsip-0073.md) | Match Force-Settlement Orders with Margin Calls and Limit Orders | Abit More | Protocol | Draft
-[74](bsip-0074.md) | Margin Call Fee Ratio | Jerry Liu | Protocol | Draft
-[75](https://github.com/bitshares/bsips/issues/96) | Asset Owner Defines MCR and MSSR Values | John Jones | Protocol | Draft
+[69](bsip-0069.md) | Additional Assert Predicates | Christopher J. Sanborn | Protocol | Voting
+[70](bsip-0070.md) | Peer-to-Peer Leveraged Trading | George Harrap, Michel Santos, Peter Conrad | Protocol | Voting
+[71](bsip-0071.md) | Add "Prevent Global Settlement" Flag for Smartcoin  | Jerry Liu | Protocol | Voting
+[72](bsip-0072.md) | Tanks and Taps: A General Solution for Smart Contract Asset Handling | Nathan Hourt | Protocol | Voting
+[73](bsip-0073.md) | Match Force-Settlement Orders with Margin Calls and Limit Orders | Abit More | Protocol | Voting
+[74](bsip-0074.md) | Margin Call Fee Ratio | Jerry Liu | Protocol | Voting
+[75](https://github.com/bitshares/bsips/issues/96) | Asset Owner Defines MCR and MSSR Values | John Jones | Protocol | Voting
```
Comment (abitmore) [inline README.md]: Already voting? Please merge the draft into master. If already done, please update the link.

**Comment by ryanRfox** [comment]
Replaced by #233 and future PR for voting

**Pull request closed without merging**